.prettierrc

```diff
@@ -1,9 +1,20 @@
 {
 	"plugins": [
+		"@trivago/prettier-plugin-sort-imports",
 		"prettier-plugin-brace-style",
 		"prettier-plugin-space-before-function-paren",
 		"prettier-plugin-merge"
 	],
+	"importOrder": [
```

Comment:
How does this work? What order is it specifying?

Reply:
Added comments in this PR: https://github.com/PrismJS/prism/pull/3917